app/assets/stylesheets/publisher.scss

```diff
@@ -29,25 +29,37 @@
     }
 
     .options_and_submit {
-      padding: 10px 0 ;
-      #publisher_service_icons {
-        .btn-link{ text-decoration: none; }
-        .btn-link.question_mark .entypo-cog {
+      padding: 10px 0;
+      .btn{ height: 32px; }
```

Comment:
![button height](https://cloud.githubusercontent.com/assets/3798614/9831475/7382afd4-5959-11e5-8678-ffe5890026ad.png)

There is now less white space between the text and the bottom border.